package.json

```diff
@@ -7,7 +7,8 @@
     "build": "webpack",
     "start": "npm run build -- --stats=errors-only && electron dist",
     "start-local": "npm run build -- --stats=errors-only && electron dist --control.address=http://localhost:4444 --control.username=streamwall --control.password=local-dev",
-    "test": "jest"
+    "test-full": "jest",
+    "test": "jest --ci --reporters=default --reporters=jest-junit --testPathIgnorePatterns=src/node/server.test.js --coverage"
```

Comment:
I'd prefer that the standard interactive `jest` script was named `jest` and the CI one was named something specific like `test:ci`.